src/main/java/cms/gov/madie/measure/services/BundleService.java

```diff
@@ -21,30 +21,31 @@ public class BundleService {
 
   private final ElmTranslatorClient elmTranslatorClient;
 
-  public String bundleMeasure(Measure measure, String accessToken) {
+  public String getMeasureBundleForCalculation(Measure measure, String accessToken) {
     if (measure == null) {
       return null;
     }
 
     try {
       retrieveElmJson(measure, accessToken);
-      return fhirServicesClient.getMeasureBundle(measure, accessToken);
+      return fhirServicesClient.getMeasureBundleForCalculation(measure, accessToken);
     } catch (RestClientException | IllegalArgumentException ex) {
       log.error("An error occurred while bundling measure {}", measure.getId(), ex);
       throw new BundleOperationException("Measure", measure.getId(), ex);
     }
   }
 
-  public ResponseEntity<byte[]> exportBundleMeasure(Measure measure, String accessToken) {
+  public ResponseEntity<String> getMeasureBundleForExport(Measure measure, String accessToken) {
```

Comment:
This comment isn't directly related to your change, so we can offline it if needed.

Since this is in a Service and not a Controller, it's odd it returns a ResponseEntity. Any idea if it's critical or can it be changed to some POJO?

Reply:
@jkotanchik-SB Yeah I can change it to String.